Formula/inetutils.rb

```diff
@@ -28,60 +36,88 @@ def noshadow
     list
   end
 
+  def linux_conflicts
+    # List of binaries that conflict with other common implementations
+    list = %w[dnsdomainname hostname ifconfig] # net-tools
+    list << "logger" # util-linux
+    list << "ping" # iputils
+    list << "whois" # whois
+    list
+  end
+
   def install
-    args = %W[
-      --disable-dependency-tracking
-      --disable-silent-rules
-      --prefix=#{prefix}
-      --with-idn
-    ]
-
-    args << "--program-prefix=g" if OS.mac?
-    system "./configure", *args
+    system "./configure", *std_configure_args,
+                          "--disable-silent-rules",
+                          "--with-idn",
+                          "--program-prefix=g"
     system "make", "SUIDMODE=", "install"
 
-    if OS.mac?
-      # Binaries not shadowing macOS utils symlinked without 'g' prefix
-      noshadow.each do |cmd|
-        bin.install_symlink "g#{cmd}" => cmd
-        man1.install_symlink "g#{cmd}.1" => "#{cmd}.1"
-      end
+    no_conflict = OS.mac? ? noshadow : []
 
-      # Symlink commands without 'g' prefix into libexec/gnubin and
-      # man pages into libexec/gnuman
-      bin.find.each do |path|
+    # Symlink server commands without 'g' prefix into sbin on Linux.
+    # (ftpd, inetd, rexecd, rlogind, rshd, syslogd, talkd, telnetd, tftpd, uucpd)
+    if OS.linux?
+      libexec.find.each do |path|
```

Comment:
By default, these are installed into `libexec`. Arch uses `--libexec=/usr/bin` to install into `bin` while Debian chooses to install into `sbin`.

For now, just symlinked Linux's `libexec/g{ftp,inet,...}d` executables into `sbin`. Wasn't sure about macOS since they could shadow system binaries and/or conflict with other formulae (e.g. `telnetd`).